x/auth/vesting/types/vesting_account_test.go

```diff
@@ -62,43 +62,87 @@ func TestGetVestedCoinsContVestingAcc(t *testing.T) {
 
 func TestGetVestingCoinsContVestingAcc(t *testing.T) {
 	now := tmtime.Now()
-	endTime := now.Add(24 * time.Hour)
+	// start-time in the future:
+	startTime := now.Add(24 * time.Hour)
+	endTime := startTime.Add(24 * time.Hour)
 
 	bacc, origCoins := initBaseAccount()
-	cva := types.NewContinuousVestingAccount(bacc, origCoins, now.Unix(), endTime.Unix())
+	cva := types.NewContinuousVestingAccount(bacc, origCoins, startTime.Unix(), endTime.Unix())
 
 	// require all coins vesting in the beginning of the vesting schedule
 	vestingCoins := cva.GetVestingCoins(now)
 	require.Equal(t, origCoins, vestingCoins)
 
+	// require all coins vesting before right start time of the vesting schedule
+	vestingCoins = cva.GetVestingCoins(startTime.Add(-1))
+	require.Equal(t, origCoins, vestingCoins)
+
 	// require no coins vesting at the end of the vesting schedule
 	vestingCoins = cva.GetVestingCoins(endTime)
 	require.Nil(t, vestingCoins)
 
 	// require 50% of coins vesting
-	vestingCoins = cva.GetVestingCoins(now.Add(12 * time.Hour))
+	vestingCoins = cva.GetVestingCoins(startTime.Add(12 * time.Hour))
 	require.Equal(t, sdk.Coins{sdk.NewInt64Coin(feeDenom, 500), sdk.NewInt64Coin(stakeDenom, 50)}, vestingCoins)
+
+	// require 25% of coins vesting
+	vestingCoins = cva.GetVestingCoins(startTime.Add(18 * time.Hour))
+	require.Equal(t, sdk.Coins{sdk.NewInt64Coin(feeDenom, 250), sdk.NewInt64Coin(stakeDenom, 25)}, vestingCoins)
+}
+
+func TestGetVestingCoinsContVestingAccStartTimeInFuture(t *testing.T) {
```

Comment:
[question] how does this test differ from the test above titled: `TestGetVestingCoinsContVestingAcc` which uses a start time in the future?

Reply:
I could swear I deleted this one and just used the existing. Great catch. Fixed in [1fb01d7](https://github.com/celestiaorg/cosmos-sdk/pull/342/commits/1fb01d7fdede6237c18d33ae2575d438f61f5358)